Dockerfile

```diff
@@ -261,5 +261,7 @@ ENV LOCALSTACK_BUILD_VERSION=${LOCALSTACK_BUILD_VERSION}
 # expose edge service, external service ports, and debugpy
 EXPOSE 4566 4510-4559 5678
 
+HEALTHCHECK --interval=10s --start-period=15s --retries=5 --timeout=5s CMD curl --fail http://localhost:4566/health || exit 1
+
```

Comment:
Unfortunately, this won't work if anyone changes the edge port (by setting the `EDGE_PORT` env).
Imho, we should use the CLI instead (which should use the correct configuration). It allows checking the service health with `localstack status services` and returns a non-zero exit code (`1`) in case the health endpoint cannot be reached. You can directly use the CLI after activating the virtual environment.